CMakeLists.txt

```diff
@@ -88,6 +89,11 @@ else()
 endif()
 message( VERBOSE "AMDGPU_TARGETS=${AMDGPU_TARGETS}")
 
+if(NOT ${CURRENT_GPU} IN_LIST AMDGPU_TARGETS)
```

Comment:
you will probably have to loop through each of the AMDGPU_TARGETS and do a partial string match for the value in CURRENT_GPU. Our targets will have the xnack flag appended.